lib/apus/adapters/twilio_adapter.ex

```diff
@@ -20,7 +20,9 @@ defmodule Apus.TwilioAdapter do
         message = %Apus.Message{
           from: body["from"],
           to: body["to"],
-          body: body["body"]
+          body: body["body"],
+          provider: "Twilio",
```

Comment:
I would suggest lowercasing this to `twilio` for ease of future comparison by callers of the library.